tools/introspection.c

```diff
@@ -568,6 +578,7 @@ hijack_threads(struct ulp_process *process)
   struct ulp_thread *t;
 
   DEBUG("entering the critical section (process hijacking).");
+  process->all_threads_hijacked = FALSE;
```

Comment:
Even though I believe that this initialization works, I think it's misplaced. The actual initialization should be closer to the declaration of `process`, i.e. in the trigger, check, and ulp tools. Currently, the initialization is implicit in the zeroing of the whole struct in, for instance:

https://github.com/SUSE/libpulp/blob/2e790a15612c5afc2db0d48e7fc335097963d7a8/tools/trigger.c#L163

Reply:
By the way, I don't think this change is wrong, nor that it needs to go away. I just wanted to make sure we don't treat this as the initialization.

Reply:
Are you saying then to add an explicit initialization to `false` after memset? Happy to do that.